src/map_io/tribes_legacy_lookup_table.cc

```diff
@@ -38,7 +38,12 @@ TribesLegacyLookupTable::TribesLegacyLookupTable()
         {"reed_small", "reedfield_small"},
         {"reed_tiny", "reedfield_tiny"},
         {"reed_ripe", "reedfield_ripe"},
-     } {
+     },
+	 // Buildings
+     buildings_{},
+	 // Ships
+	 ships_{}
+     {
 }
```

Comment:
Are you sure about this opening bracket? Remove the closing on instead.

Reply:
actually I am quite sure since those are the opening/closing brackets of the constructor body after the initialization list (which is a bit long)  ;)

Reply:
You're completely right. Should check my 👓  😃